third_party/terraform/resources/resource_dataproc_cluster.go.erb

```diff
@@ -674,6 +692,14 @@ func expandAutoscalingConfig(cfg map[string]interface{}) *dataproc.AutoscalingCo
 	}
 	return conf
 }
+
+func expandEndpointConfig(cfg map[string]interface{}) *dataproc.EndpointConfig {
+	conf := &dataproc.EndpointConfig{}
```

Comment:
I think you'll also want set `ForceSendFields: []string{"EnableHttpPortAccess"}` in order to guarantee that false values get sent

Reply:
Turns out the field is only available on create, meaning no need for this